packages/K4FWCore/package.py

```diff
@@ -1,95 +1,47 @@
-##############################################################################
-# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
-# Produced at the Lawrence Livermore National Laboratory.
-#
-# This file is part of Spack.
-# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
-# LLNL-CODE-647188
-#
-# For details, see https://github.com/llnl/spack
-# Please also see the LICENSE file for our notice and the LGPL.
-#
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU Lesser General Public License (as
-# published by the Free Software Foundation) version 2.1, February 1999.
-#
-# This program is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
-# conditions of the GNU Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public
-# License along with this program; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
-##############################################################################
-
 from spack import *
-import llnl.util.tty as tty
-import subprocess
 
 class K4fwcore(CMakePackage):
     """Core framework components of the Key4HEP project"""
     homepage = "https://github.com/key4hep/K4FWCore"
+    url = "https://github.com/key4hep/K4FWCore/archive/v00-01.tar.gz"
     git = "https://github.com/key4hep/K4FWCore.git"
 
     version('master', branch='master')
-    version('develop', branch='master')
+    version('0.1.0', sha256='05326d0f3d222f4a195baebf9c9fc60651621ec293e4384de3aaa81281cbea7a')
 
     variant('build_type', default='Release',
             description='CMake build type',
             values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
 
-    variant('lcg', default=True, description="Installed against an LCG Release")
     variant('cxxstd',
             default='17',
             values=('14', '17'),
             multi=False,
             description='Use the specified C++ standard when building.')
 
-
-
-    depends_on('gaudi')
-    depends_on('root')
-    depends_on('podio')
-
-
-    depends_on('vdt', when="+lcg")
-    depends_on('python', when="+lcg")
-    depends_on('davix', when="+lcg")
-
+    depends_on('gaudi@32.2:')
+    depends_on('root@6.08:')
+    depends_on('podio@0.10:')
 
     def cmake_args(self):
         args = []
         # C++ Standard
         args.append('-DCMAKE_CXX_STANDARD=%s' % self.spec.variants['cxxstd'].value)
+        # not really used for spack builds but needs to be set
+        args.append('-DHOST_BINARY_TAG=x86_64-linux-gcc9-opt')
```

Comment:
Does this cause problems when using gcc8? 
Add an if for gaudi below 34, as it shouldn't be needed for gaudi 34 onwards?

Reply:
No, I built this recipe with gcc8, it seems to be a legacy thing. As far as I can tell it is not used if plain CMake is used, but needs to be a certain format otherwise errors are throwsn.

Reply:
It should not be needed and will just be ignored in v34.

Reply:
Still would be good to add a comment why this is here, and for which versions of gaudi.

Reply:
That's true, done. See also here: https://gitlab.cern.ch/gaudi/Gaudi/-/issues/123#note_3416458